h/security/policy/top_level.py

```diff
@@ -59,6 +59,15 @@ def get_subpolicy(request):
     )
     api_authcookie = api_authcookie.bind(request)
 
+    if is_api_request(request):
+        return APIPolicy(
+            [
+                BearerTokenPolicy(),
+                AuthClientPolicy(),
+                APICookiePolicy(api_authcookie, AuthTicketCookieHelper()),
+            ]
+        )
```

Comment:
This is just reverting https://github.com/hypothesis/h/pull/8913. The diff is slightly confusing because https://github.com/hypothesis/h/pull/8913 also had to move the code, but the only real difference is that `api_authcookie` is now passed to `APICookiePolicy()` rather than `html_authcookie`.